requirements.txt

```diff
@@ -22,6 +22,7 @@ paho-mqtt==1.6.1
 Pillow>=5.3
 psycopg2-binary==2.9.9
 PyJWT~=2.8.0
+respx==0.22.0
```

Comment:
Due to our multistage docker build, any dependency changes need to have the base image rebuilt and pushed to GHCR, you should have write access, set up a github PAT (see `.envrc.example`), then run this on your branch and commit the result:

```sh
make docker-login
make build-base-docker-image
```